c/expr/fbinary/math.cc

```diff
@@ -58,10 +92,102 @@ The length of the hypotenuse of a right triangle with sides x and y.
 Equivalent to ``sqrt(x*x + y*y)``.
 )";
 
-static
 py::PKArgs args_hypot(2, 0, 0, false, false, {"x", "y"}, "hypot", doc_hypot);
 
 
+template <typename T>
+static bimaker_ptr _hypot(SType uptype1, SType uptype2, SType outtype) {
+  return bimaker1<T, T, T>::make(std::hypot, uptype1, uptype2, outtype);
+}
+
+
+bimaker_ptr resolve_fn_hypot(SType stype1, SType stype2) {
+  SType uptype1, uptype2;
+  SType stype0 = _resolve_math_stypes(stype1, stype2, &uptype1, &uptype2);
+  switch (stype0) {
+    case SType::FLOAT32:  return _hypot<float>(uptype1, uptype2, stype0);
+    case SType::FLOAT64:  return _hypot<double>(uptype1, uptype2, stype0);
+    default:
+      throw TypeError() << "Cannot apply function `hypot()` to columns with "
+          "types `" << stype1 << "` and `" << stype2 << "`";
+  }
+}
+
+
+
+
+//------------------------------------------------------------------------------
+// Op::POWERFN
+//------------------------------------------------------------------------------
+
+static const char* doc_pow =
+R"(pow(x, y)
+--
+
+Number x raised to the power y. The return value will be float, even
+if the arguments x and y are integers.
+
+This function is equivalent to `x ** y`.
+)";
+
+py::PKArgs args_pow(2, 0, 0, false, false, {"x", "y"}, "pow", doc_pow);
+
+
+template <typename T>
+static bimaker_ptr _pow(SType uptype1, SType uptype2, SType outtype) {
+  return bimaker1<T, T, T>::make(std::pow, uptype1, uptype2, outtype);
+}
+
+
+bimaker_ptr resolve_fn_pow(SType stype1, SType stype2) {
+  SType uptype1, uptype2;
+  SType stype0 = _resolve_math_stypes(stype1, stype2, &uptype1, &uptype2);
+  switch (stype0) {
+    case SType::FLOAT32:  return _pow<float>(uptype1, uptype2, stype0);
+    case SType::FLOAT64:  return _pow<double>(uptype1, uptype2, stype0);
+    default:
+      throw TypeError() << "Cannot apply function `pow()` to columns with "
+          "types `" << stype1 << "` and `" << stype2 << "`";
+  }
+}
+
+
+
+
+//------------------------------------------------------------------------------
+// Op::COPYSIGN
+//------------------------------------------------------------------------------
+
+static const char* doc_copysign =
+R"(copysign(x, y)
+--
+
+Return a float with the magnitude of x and the sign of y.
+)";
+
+py::PKArgs args_copysign(2, 0, 0, false, false, {"x", "y"}, "copysign",
+                         doc_copysign);
+
+
+template <typename T>
+static bimaker_ptr _copysign(SType uptype1, SType uptype2, SType outtype) {
+  return bimaker1<T, T, T>::make(std::copysign, uptype1, uptype2, outtype);
```

Comment:
On Windows `copysign` is not a member of `std`. Just leaving this note here to take care.

Reply:
Really? It's in C++11: https://en.cppreference.com/w/cpp/numeric/math/copysign, and in C99 too: https://en.cppreference.com/w/c/numeric/math/copysign

Reply:
Yeah, I saw that, but not in MSVC-world https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/copysign-copysignf-copysignl-copysign-copysignf-copysignl?view=vs-2019

Reply:
Btw, it could be compiler dependent. At least in 15.9.17 I'm getting
```
error C2039: '_copysign': is not a member of 'std'
```
In 19.00.23506 there is no such message. I assume that the first version is VS 2017, the second one is VS 2019.

Reply:
Moved to MSVC 2019 and this one has the `std::copysign` problem fixed. Marking this as resolved.